### PR TITLE
[PHP 8.0] Make use of match()

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Ready for review :+1:  Just rebased, otherwise passing

File: app/bundles/EmailBundle/MonitoredEmail/Mailbox.php [hunk 1]
```diff
@@ -935,14 +935,11 @@ protected function initMailPart(Message $mail, $partStructure, $partNum, $markAs
...
-                                break;
-                            case 'html':
-                            default:
-                                $mail->textHtml .= $data;
-                        }
+                        match ($subtype) {
+                            'plain' => $mail->textPlain .= $data,
+                            // no break
+                            default => $mail->textHtml .= $data,
+                        };
```
Comment (escopecz) [inline app/bundles/EmailBundle/MonitoredEmail/Mailbox.php:942]: Does the comment make sense here?

**Review by escopecz — DISMISSED** [review]
Nice! Again, reads much better. One small question.

**Comment by TomasVotruba** [comment]
I've just rebased btw

File: app/bundles/EmailBundle/MonitoredEmail/Mailbox.php (hunk 1 above)
Comment (TomasVotruba) [reply app/bundles/EmailBundle/MonitoredEmail/Mailbox.php:942]: Not really, I'll remove it :+1:

**Comment by TomasVotruba** [comment]
Good to go :+1:

**Review by escopecz — APPROVED** [review]
Thank you! 👍

**Pull request merged**